### ISSUE TITLE
Invisible minimize button appears after length and width have been reduced

### ISSUE THREAD

**Issue by Qantex** [microsoft/terminal#5747]
# Environment

```none
Windows build number: 10.0.18363.815
Windows Terminal version (if applicable): 0.11.1191.0

Any other software?
```

# Steps to reproduce

- Start Windows Terminal.
- Reduce the length and width until it is no longer possible.
- Now move the mouse pointer to the title bar.
- Now move the mouse pointer to the left until the mouse pointer has a small distance to Windows Terminal.

# Expected behavior
**Windows Terminal should not show an invisible minimize button**

# Actual behavior
**Windows Terminal displays an invisible button for minimizing.**

**Comment by zadjii-msft** [comment]
Could you maybe post a screenshot of this when it's reproing? Maybe something got lost in translation, but I can't get this to repro locally:

![image](https://user-images.githubusercontent.com/18356694/81065346-e79da000-8ea0-11ea-97d5-ce053bf81b1f.png)

(Imagine that poorly drawn cursor is my mouse cursor)

**Comment by Qantex** [comment]
@zadjii-msft You have performed the steps correctly, but the distance between the mouse pointer and the Windows terminal is too great.
![Anmerkung 2020-05-05 150715](https://user-images.githubusercontent.com/47790579/81069798-aded8900-8ee2-11ea-820b-c6860280f138.jpg)

**Comment by Qantex** [comment]
@zadjii-msft You have forgotten to reduce the length and width until this is no longer possible.

**Comment by Qantex** [comment]
@zadjii-msft You have forgotten to reduce the length and width, reduce the length and width until this is no longer possible and the distance between the mouse pointer and the Windows terminal is too great.

![Anmerkung 2020-05-05 150715](https://user-images.githubusercontent.com/47790579/81071121-85ff2500-8ee4-11ea-91bb-e1fdfef405db.jpg)

**Comment by zadjii-msft** [comment]
Ah okay, this is actually fixed in #5599, which I guess wasn't in 0.11.1191. Once that PR rolls out, you won't actually be able to make the Terminal window smaller than that size in my screenshot, so this bug will be impossible to hit.

/dup #4976

**Comment by Qantex** [comment]
@zadjii-msft Many thanks for your help!
